.travis.yml

```diff
@@ -0,0 +1,18 @@
+language: java
+jdk:
+  - oraclejdk8
+
+before_install:
+ - chmod +x gradlew
+
+script:
+  - jdk_switcher use openjdk8
+  - ./gradlew clean check
+
+before_cache:
+  - rm -f  $HOME/.gradle/caches/modules-2/modules-2.lock
+  - rm -fr $HOME/.gradle/caches/*/plugin-resolution/
+cache:
+  directories:
+    - $HOME/.gradle/caches/
+    - $HOME/.gradle/wrapper/
```

Comment:
On success publish to github pages. See https://github.com/concordion/concordion-screenshot-extension-demo/blob/master/.travis.yml